src/d3a/models/market/__init__.py

```diff
@@ -189,9 +189,16 @@ def __repr__(self):  # pragma: no cover
     @staticmethod
     def sorting(offers_bids: Dict, reverse_order=False) -> List[Union[Bid, Offer]]:
         """Sort a list of bids or offers by their energy_rate attribute."""
-        return sorted(offers_bids.values(),
-                      key=lambda obj: obj.energy_rate,
-                      reverse=reverse_order)
+        if reverse_order:
+            # Sorted bids in descending order
+            return list(reversed(sorted(
+                offers_bids.values(),
+                key=lambda obj: obj.energy_rate)))
+        else:
+
+            return sorted(offers_bids.values(),
+                          key=lambda obj: obj.energy_rate,
+                          reverse=reverse_order)
```

Comment:
Shall we reuse `sort_list_of_dicts_by_attribute` here in oder to be DRY?

Reply:
The data structure here is different (dict of lists as values) whereas the mentioned function deals with list of dicts
However this is possible, I can do it in the ticket I'm currently working on ( supporting 1-N bids/offers matching) if you agree